models/intermediate/int_zuora__transactions_grouped.sql

```diff
@@ -18,8 +18,8 @@ transactions_grouped as (
         sum(invoice_items) as daily_invoice_items,
 
         {% for col in sum_cols %}
-        {% if var('using_multicurrency', true) %}
-            sum({{ col }}_home_currency) as daily_{{ col }},
+        {% if var('using_multicurrency', true) %} 
+            sum({{ col }}_home_currency) as daily_{{ col }}, 
```

Comment:
Sorry, a bit confused with variable configuration here--isn't the if loop just checking the `dbt_project.yml` for the variables and picking and choosing based on what condition is holding? Because it'd move to the false condition if this is false. 

Or are we saying our variables should be set in our loops to our Readme recommendations as the first parameter?